src/App.css

```diff
@@ -44,6 +52,83 @@
   border-radius: 0.3rem;
 }
 
```

Comment:
- [ ] In the navbar must be a profile icon that is missing in your design. Please add it. 😄 👍 

![tempsnip](https://user-images.githubusercontent.com/64173189/156856856-bb586ebd-35bf-42b8-8328-2c6e64d5ca39.png)

Reply:
Thank you! I have added a profile icon.